debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -406,15 +454,14 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
                                                            .withDependents(DATABASE_WHITELIST_NAME)
                                                            .withDescription("Flag specifying whether built-in tables should be ignored.");
 
-    public static final Field JDBC_DRIVER = Field.create("database.jdbc.driver")
-                                                .withDisplayName("Jdbc Driver Class Name")
-                                                .withType(Type.CLASS)
-                                                .withWidth(Width.MEDIUM)
-                                                .withDefault(com.mysql.jdbc.Driver.class.getName())
-                                                .withImportance(Importance.LOW)
-                                                .withValidation(Field::isClassName)
-                                                .withDescription("JDBC Driver class name used to connect to the MySQL database server.");
-    /**
+    public static final Field VENDOR = Field.create("mysql.vendor")
```

Comment:
What's the reasoning for introducing this over the existing "driver" property?

We could still use the new enum internally, parsed from the given driver class name. I'd prefer though to not break compatability with existing users of the driver option.

Reply:
There could not be any users of driver property as it goes hand-in-hand with hard-coded JDBC url. So albeit this is superficially a configrable value it is effectivelly a constant. And as JDBC URL pattern and driver must be changed in accord I thought it is best to replace it with a single logical constant.